vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextArea.java

```diff
@@ -216,16 +218,12 @@ public void setInvalid(boolean invalid) {
         super.setInvalid(invalid);
     }
 
-    @Override
-    public void setLabel(String label) {
-        super.setLabel(label);
-    }
```

Comment:
This method was not removed in the original PR. Doing so might break binary compatibility, so let's keep it.